.github/workflows/post-release.yaml

```diff
@@ -0,0 +1,14 @@
+# Toggle webhook to pull latest release onto dl.pelicanplatform.org
```

Comment:
Can you make this comment a little more verbose and explain what sending a post request to that endpoint does? I tried the curl command locally and can't figure out what's happening other than that it spits out `{"message":"success"}` lol

Reply:
How is this?

# Toggle webhook to pull latest release onto pelicanplatform.org and update the download offerings there

Hitting this endpoint toggles dl.pelicanplatform.org to update. Which post release means pulling in the new release and updating the symlinks accordingly.

Reply:
```
# Toggle webhook to pull latest release onto pelicanplatform.org and update the download offerings there
# Post release this will result in the new release being available and the Major/Minor pointers being moved/created accordingly
```